packages/tocco-ui/src/EditableValue/typeEditors/SearchFilterEdit.js

```diff
@@ -1,19 +1,14 @@
+import React from 'react'
 import PropTypes from 'prop-types'
-import typeEditorFactory from './../typeEditorFactory'
+import MultiSelect from './MultiSelect'
+import SingleSelect from './SingleSelect'
 
 const SearchFilterEdit = props => {
-  const type = props.options.multi ? 'multi-select' : 'single-select'
-  return (
-    typeEditorFactory(
-      type,
-      props.value,
-      props.onChange,
-      props.options,
-      props.id,
-      props.events,
-      props.readOnly
-    )
-  )
+  if (props.options.multi) {
```

Comment:
`options` is not a required prop. There should either be a null check here or the prop be declared as required.